tests/unit/test_models/standard_output_tests.py

```diff
@@ -0,0 +1,448 @@
+#
+# Standard tests on the standard set of model outputs
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+
+import pybamm
+import numpy as np
+
+
+class StandardOutputTests(object):
+    "Calls all the tests on the standard output variables."
+
+    def __init__(self, model, disc, solver, parameter_values):
+        self.model = model
+        self.disc = disc
+        self.solver = solver
+
+        if isinstance(self.model, pybamm.LithiumIonBaseModel):
+            self.chemistry = "Lithium-ion"
+        elif isinstance(self.model, pybamm.LeadAcidBaseModel):
+            self.chemistry = "Lead acid"
+
+        current_sign = np.sign(parameter_values["Typical current density"])
+        if current_sign == 1:
+            self.operating_condition = "discharge"
+        elif current_sign == -1:
+            self.operating_condition = "charge"
+        else:
+            self.operating_condition = "off"
+
+    def test_voltage(self):
+        tests = VoltageTests(
+            self.model, self.disc, self.solver, self.operating_condition
+        )
+        tests.test_all()
+
+    def test_electrolyte(self):
+        tests = ElectrolyteConcentrationTests(
+            self.model, self.disc, self.solver, self.operating_condition
+        )
+        tests.test_all()
+
+    def test_potentials(self):
+        tests = PotentialTests(
+            self.model, self.disc, self.solver, self.operating_condition
+        )
+        tests.test_all()
+
+    def test_particles(self):
+        tests = ParticleConcentrationTests(
+            self.model, self.disc, self.solver, self.operating_condition
+        )
+        tests.test_all()
+
+    def test_currents(self):
+        tests = CurrentTests(
+            self.model, self.disc, self.solver, self.operating_condition
+        )
+        tests.test_all()
+
+    def test_all(self):
```

Comment:
This is a very good way of splitting up the tests! You could remove some redundant code with something like 
```python3
def run_tests(self, class_name):
     tests = class_name(self.model, ...)
     tests.test_all 

def test_all(self):
     self.run_class(VoltageTests)
     etc
```
Also, the two `test_all` functions doing different things might be confusing. Maybe call this one `run_all_output_tests` or something

Reply:
Yeah I agree, given that this really isn't an OO bit of code it's probably best to just replace the class by a function. That should cut out alot of unneeded extra code. It's a little out of style with the rest of the tests though which is why I made a class

Reply:
I think it's fine to leave `VoltageTests` etc as classes, but you don't have to write a separate function like `test_voltage` for each one. Made a typo above, what I meant was:
```python3
def run_class(self, class_name):
     tests = class_name(self.model, ...) # <--  here class_name can be any class like `VoltageTests`, `ParticleConcentrationTests` etc
     tests.test_all()

def test_all(self):
     self.run_class(VoltageTests)
     etc
```